### PR TITLE
Make PHPCS happy

### PR DESCRIPTION
**Pull request by rebeccahum**
## Description
Should be no functional changes, just whitespace and syncing lint fixes from upstream and adding phpcs:ignore comments

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [ ] This change has the fix PRed upstream (if applicable). If not applicable, it has the relevant "// VIP: reason for the discrepancy with upstream" comment in places where the code is discrepant.

### CODE REVIEW

File: includes/classes/Feature/Facets/Facets.php [hunk 1]
```diff
@@ -470,7 +470,7 @@ public function build_query_url( $filters ) {
 		 */
 		$query_string = apply_filters( 'ep_facet_query_string', $query_string, $query_param );
 
-		$url        = $_SERVER['REQUEST_URI'];
+		$url        = isset( $_SERVER['REQUEST_URI'] ) ? sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ) : '';
```
Comment (sjinks) [inline includes/classes/Feature/Facets/Facets.php:473]: I am not 100% sure about this change.

Comment (rebeccahum) [reply includes/classes/Feature/Facets/Facets.php:473]: @sjinks This change is pulled from upstream

File: includes/classes/Screen.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ public function determine_screen() {
 		// VIP: We removed the block about returning false depending on network admin or in network mode && not network admin
 
 		// phpcs:disable WordPress.Security.NonceVerification
-		if ( ! empty( $_GET['page'] ) && false !== strpos( $_GET['page'], 'elasticpress' ) ) {
+		if ( ! empty( $_GET['page'] ) && false !== strpos( sanitize_key( $_GET['page'] ), 'elasticpress' ) ) {
```
Comment (sjinks) [inline includes/classes/Screen.php:59]: While this works, the call to `sanitize_key()` incurs some performance penalty. If we only check for a substring, I would not call `sanitize_key()`.

File: includes/dashboard.php [hunk 3]
```diff
@@ -189,7 +189,7 @@ function maybe_skip_install() {
 		return;
 	}
 
-	if ( empty( $_GET['ep-skip-install'] ) || empty( $_GET['nonce'] ) || ! wp_verify_nonce( $_GET['nonce'], 'ep-skip-install' ) || ! in_array( Screen::factory()->get_current_screen(), [ 'install' ], true ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+	if ( empty( $_GET['ep-skip-install'] ) || empty( $_GET['nonce'] ) || ! wp_verify_nonce( sanitize_key( $_GET['nonce'] ), 'ep-skip-install' ) || ! in_array( Screen::factory()->get_current_screen(), [ 'install' ], true ) ) { // phpcs:ignore WordPress.Security.NonceVerification
```
Comment (sjinks) [inline includes/dashboard.php:192]: The same is true here: `wp_verify_nonce()` will return `false` for an invalid nonce; it makes little sense to corrupt an invalid nonce with `sanitize_key()`.

File: includes/dashboard.php [hunk 4]
```diff
@@ -235,6 +235,7 @@ function maybe_clear_es_info_cache() {
 
 	if ( ! empty( $_GET['ep-retry'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
 		wp_safe_redirect( remove_query_arg( 'ep-retry' ) );
+		exit();
```
Comment (sjinks) [inline includes/dashboard.php:238]: Nice catch

File: includes/dashboard.php (hunk 3 above)
Comment (rebeccahum) [reply includes/dashboard.php:192]: @sjinks This was also pulled from upstream

File: includes/partials/settings-page.php [hunk 5]
```diff
@@ -37,7 +37,8 @@
 <div class="wrap">
 	<h1><?php esc_html_e( 'Settings', 'elasticpress' ); ?></h1>
 
-	<form action="<?php echo esc_attr( $action ); ?>" method="post" class="ep-settings">
+	<form action="
+	<?php echo esc_attr( $action ); // phpcs:ignore WordPressVIPMinimum.Security.ProperEscapingFunction.hrefSrcEscUrl ?>" method="post" class="ep-settings">
```
Comment (sjinks) [inline includes/partials/settings-page.php]: Shouldn't we use `esc_url()` here?

File: includes/classes/Screen.php (hunk 2 above)
Comment (rebeccahum) [reply includes/classes/Screen.php:59]: @sjinks This as well was pulled from upstream

File: includes/partials/settings-page.php (hunk 5 above)
Comment (rebeccahum) [reply includes/partials/settings-page.php]: Done.

**Review by sjinks — APPROVED** [review]

**Pull request merged**